hack/update-mockgen.sh

```diff
@@ -27,8 +27,7 @@ done
 # github.com/prysmaticlabs/prysm/v5/validator/client/iface
 # --------------------------------------------------------
 iface_mocks=(
-      "$iface_mock_path/beacon_chain_client_mock.go BeaconChainClient"
-      "$iface_mock_path/prysm_beacon_chain_client_mock.go PrysmBeaconChainClient"
```

Comment:
Instead of removing this, we should generate a mock for `PrysmChainClient`

```
"$iface_mock_path/prysm_chain_client_mock.go PrysmChainClient"
```

Reply:
Thanks. Done